centreon/unattended.sh

```diff
@@ -431,7 +431,8 @@ function setup_mysql() {
 		log "INFO" "Successfully installed the $dbms repository"
 	fi
 	systemctl enable --now $mysql_service_name
-	if ! [[ "$version" == "25.09" || "$version" =~ "25.1"[0-2] || "$version" =~ "26.0"[1-9] ]]; then
+
+	if ! [[ "$version" == "24.10" || "$version" == "25.09" || "$version" =~ "25.1"[0-2] || "$version" =~ "26.0"[1-9] ]]; then
```

Comment:
this change will need to be applied on develop branch
(since all versions are installed from unattended on develop branch)